packages/test-utils/Cargo.toml

```diff
@@ -6,11 +6,11 @@ version.workspace = true
 license.workspace = true
 repository.workspace = true
 authors.workspace = true
-publish           = true
+publish.workspace = true
 
 [dependencies]
-babylon-apis     = { path = "../apis", version = "0.17.0" }
-babylon-proto    = { path = "../proto", version = "0.17.0" }
+babylon-apis     = { path = "../apis" }
```

Comment:
why not `babylon-apis     = { path = "../apis", version = "v1.0.0-rc.0" }`?

Reply:
Version is not necessary here as it loads the files locally

Reply:
The version is important when you publish it to crates.io. Since we're not publishing them now due to Mauro's absence, it's fine to remove them.